syntax = "proto3";

package ProtoTransport;

message Edge {
    uint32 from = 1;
    uint32 to = 2;
    bool type = 3;
    double time = 4;
    string item_name = 5;
    int32 span_count = 6;
}

message Router {
    int32 bus_wait_time = 1;
    int32 bus_velocity = 2;
}
